CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+### Release 2.2.1 (June 28, 2019)
+[Milestone#32](https://github.com/awslabs/amazon-kinesis-client/milestone/32)
+* Add periodic logging of the the state of the thread pool executor service.
```

Comment:
Add periodic logging of the the state of the thread pool executor service, that executes the async tasks submitted to and by the ShardConsumer?

Reply:
Changed to the above